org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/IDesktop.java

```diff
@@ -215,4 +215,7 @@
 	public void setTabTitle(String title, int windowNo);
 	
 	public void renderHomeTab();
+
+	default void setPredefinedContextVariables(String predefinedVariables) {

```

Comment:
don't think we have to add this to the interface. it is not a public api, just something use internally.